packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -226,3 +222,13 @@ module.exports = {
     tls: 'empty'
   }
 };
+
+// @remove-on-eject-begin
+// Override babel-loader options to not read .babelrc.
+var babelLoader = module.exports.module.loaders.find(l => l.loader === 'babel');
+babelLoader.query = babelLoader.query || {};
+babelLoader.query.babelrc = false;
+// We embed the preset instead of referring to it because otherwise
+// `env` options in the preset do not get taken into account.
```

Comment:
Hmm. It seems like `env` doesn’t work after ejecting 😢 . Then something must be wrong in how I’m using it, but I can’t figure out what.